packages-web/maps/src/utils/hooks.ts

```diff
@@ -0,0 +1,52 @@
+import { DynamicMarkersType, Marker, MarkersType, ModeledMarker } from "../../typings";
```

Comment:
I think it is better to group functionality by domain than by technology => `hooks.ts` is not the best name imho. They are basically should be together with `geocode.ts`

Reply:
Done